qcfractal/tests/test_collections.py

```diff
@@ -1368,3 +1369,32 @@ def test_get_collection_no_records_ds(fractal_compute_server):
     ds.get_entries()
     assert len(ds.data.records) == 1
     assert ds.data.records[0].name == "He1"
+
+
+def test_collection_metadata(fractal_compute_server):
```

Comment:
I usually like more tests, but this seems like a lot of volume to test one field?

Reply:
Fair enough, how about now?